tests/jeos/rpi_bluetooth.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright 2015-2021 SUSE LLC
```

Comment:
```suggestion
# Copyright SUSE LLC
```

see https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md?plain=1#L49

The same goes for other files added or changed in this pull request